docs/src/visualizations.md

```diff
@@ -318,3 +318,24 @@ j = 2 # the dimension of the plane
 
 interactive_poincaresos_scan(trs, j; linekw = (transparency = true,))
 ```
+
+## Interactive projected dynamical system
```

Comment:
Sorry, to be clear: this works for any 2D system, right? This should be 2D maps, projected 2D systems, poincare maps, whatever. Right? If so, do we even need the interactive poincare map function anymore?

Additionally please rename this section to reflect the generality of the function.